Installation/Action/waar.WaardepapierenZaakAction.action.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "WaardepapierenZaakAction",
+  "$id": "https://waardepapieren.commonground.nl/action/waar.WaardepapierenZaakAction.action.json",
+  "$schema": "https://docs.commongateway.nl/schemas/Action.schema.json",
+  "version": "0.0.1",
+  "listens": [
+    "commongateway.object.created"
```

Comment:
This listens is incorrect, the event thrown by the CoreBundle->RequestService = commongateway.object.create

Reply:
This is for an other issue, this pr is only to update the repository so it will work with the latest version of the corebundle.